ci-operator/step-registry/assisted/common/setup/assisted-common-setup-commands.sh

```diff
@@ -0,0 +1,259 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ assisted common setup command ************"
+
+mkdir -p build/ansible
+cd build/ansible
+
+cat > packing-test-infra.yaml <<-EOF
+- name: Packing assisted-test-infra
+  hosts: localhost
+  collections:
+    - community.general
+  gather_facts: no
+  vars:
+    ansible_remote_tmp: ../tmp
+  tasks:
+    - name: Compress
+      community.general.archive:
+        path: ../../
+        exclude_path:
+          - ../../build
+        dest: assisted-test-infra.tgz
+EOF
+
+ansible-playbook packing-test-infra.yaml
+
+# Get packet | vsphere configuration
+# shellcheck source=/dev/null
+set +e
+source "${SHARED_DIR}/packet-conf.sh"
+source "$SHARED_DIR/ci-machine-config.sh"
+set -e
+
+# shellcheck disable=SC2034
+export CI_CREDENTIALS_DIR=/var/run/assisted-installer-bot
+touch ${SHARED_DIR}/assisted-additional-config
+
+# TODO: Remove once OpenShift CI will be upgraded to 4.2 (see https://access.redhat.com/articles/4859371)
+~/fix_uid.sh
+
+cat << EOF > inventory
+[all]
+${IP} ansible_user=root ansible_ssh_user=root ansible_ssh_private_key_file=${SSH_KEY_FILE} ansible_ssh_common_args="-o ConnectTimeout=5 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ServerAliveInterval=90 -o LogLevel=ERROR"
+EOF
+
+cat << EOF > config.sh.j2
```

Comment:
can't we save it as file and not to create template on the fly?

Reply:
https://coreos.slack.com/archives/CBN38N3MW/p1647358663473279?thread_ts=1647357419.811569&cid=CBN38N3MW